docs/types/value-types.rst

```diff
@@ -39,6 +39,11 @@ Operators:
 * Shift operators: ``<<`` (left shift), ``>>`` (right shift)
 * Arithmetic operators: ``+``, ``-``, unary ``-``, ``*``, ``/``, ``%`` (modulo), ``**`` (exponentiation)
 
+.. warning::
+
+  Integers in Solidity are restricted to a certain range. For example, with ``uint256``, this is ``0`` up to ``2**256 - 1``.
+  If the result of some operation on those numbers does not fit inside this range, it is truncated. These truncations can have
+  `serious consequences <https://en.bitcoin.it/wiki/Value_overflow_incident>`_, and code such as ``require((balanceOf[_to] + _value) >= balanceOf[_to])`` can help you check if values are what you expect.
```

Comment:
Not sure if we should really link this wiki page....

Reply:
Perhaps we should link the safemath library?

Reply:
I think just remove it…